### PR TITLE
generate reports using rake task

### PR DESCRIPTION
**Pull request by vzaragoza**

### CODE REVIEW

File: lib/tasks/generate.rake [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    puts "Generating student and course..."
+    Generate.create_student_and_course
+    student = Generate.find_student
+    course = Generate.find_course
+
+    reports = args[:total]
+    puts "Generating #{reports} reports..."
+    Report.delete_by(student:, course:)
+    reports.times do |i|
```
Comment (mapreal19) [inline lib/tasks/generate.rake:14]: You could simplify this code with create_list and a trait. In the case this performs bad create_list might just do one SQL query

`FactoryBot.create_list(:report, reports.count, :with_fake_pdf)`

**Review by mapreal19 — COMMENTED** [review]
👍

File: lib/tasks/generate.rake (hunk 1 above)
Comment (vzaragoza) [reply lib/tasks/generate.rake:14]: I need to generate a different pdf for each report and FactoryBot always creates the same content/pdf

Comment (mapreal19) [reply lib/tasks/generate.rake:14]: Maybe with https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#sequences you can tweak it and have different contents?

**Pull request merged**